packages/codegen-ui-react/lib/forms/react-form-renderer.ts

```diff
@@ -314,8 +314,8 @@ export abstract class ReactFormTemplateRenderer extends StudioTemplateRenderer<
 
     // add model import for datastore type
     if (dataSourceType === 'DataStore') {
-      this.requiredDataModels.push(dataTypeName);
```

Comment:
I removed adding to `this.requiredDataModels` here because requiredDataModels is only used in `renderComponentOnly` and this function is not being called in `renderComponentOnly`. @lyonsbp can you double check this removal.

Reply:
yeah that should be fine

Reply:
Im working on a task that will take care of this